geopandas/tools/tests/test_sjoin.py

```diff
@@ -22,7 +22,8 @@ def setUp(self):
         nybb_filename, nybb_zip_path = download_nybb()
         self.polydf = read_file(nybb_zip_path, vfs='zip://' + nybb_filename)
         self.tempdir = tempfile.mkdtemp()
-        self.crs = {'init': 'epsg:4326'}
+        #self.crs = {'init': 'epsg:4326'}
+        self.crs = self.polydf.crs
```

Comment:
Maybe the CRS of the nybb shapefile changed when the location of the file changed, but the polydf and pointdf had different CRS values, which allowed sjoin to continue, but seemed erroneous given that pointdf's points where based off the bounding boxes of polydf. So I went ahead and changed this line to make sure pointdf used the same CRS as polydf.